.github/workflows/playwright.yml

```diff
@@ -31,6 +31,24 @@ on:
         required: false
         type: boolean
         default: false
+      docker-compose-file:
+        required: false
+        type: string
+        default: docker-compose.yml
```

Comment:
I believe this should default to an empty string rather than `docker-compose.yml`, also because the default docker-compose file scaffolded by create-plugin is called `docker-compose.yaml` (with an a):

https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/common/docker-compose.yaml

```suggestion

```